docs/functions.rst

```diff
@@ -0,0 +1,50 @@
+Useful Functions
+================
+There are some functions that interact with multiple types of objects to apply
```

Comment:
I think it'd be worth highlighting the other utility functions in the developer docs here (and directly linking, if possible).